docs/_pages/terms.md

```diff
@@ -0,0 +1,56 @@
+---

```

Comment:
We need to define our terms and policy I think. Can we check with Hari or Vinod for it?